src/templates/python_cli_entry.mjs

```diff
@@ -9,18 +9,12 @@ import { readdirSync } from "fs";
  * names it is possible this could break. The most surprising one here is tmp, I
  * am not sure why but if we link tmp then the process silently fails.
  */
-function rootDirsToMount() {
-  const skipDirs = ["dev", "lib", "proc", "tmp"];
+function dirsToMount() {
   return readdirSync("/")
-    .filter((dir) => !skipDirs.includes(dir))
+    .filter((dir) => !["dev", "lib", "proc"].includes(dir))
     .map((dir) => "/" + dir);
 }
 
-function dirsToMount() {
-  const extra_mounts = process.env["_PYODIDE_EXTRA_MOUNTS"] || "";
```

Comment:
If this variable is not used anymore, could you please remove it from the test? https://github.com/pyodide/pyodide/blob/1c29a7ce2ed29e470cf972a25df1aa86fe97da1c/src/tests/test_cmdline_runner.py#L174

Reply:
Good catch. And it's nice that we have coverage for this so we know it's not breaking anything.